.github/actions/check_if_latest_release/action.yml

```diff
@@ -0,0 +1,27 @@
+name: check_if_latest_release
+inputs:
+  token:
+    description: "Github API token"
+runs:
+  using: composite
+  steps:
+    - name: Push Bundle to MRT
```

Comment:
I think we should update the step name to something more descriptive

Reply:
Whoops, good catch. I have the wrong name there.